scan_to_inventory/static/www/models/m_product_product.js

```diff
@@ -0,0 +1,18 @@
+angular.module('scan_to_inventory').factory(
+        'ProductProductModel', [
+        '$q', '$rootScope', 'jsonRpc',
+        function ($q, $rootScope, jsonRpc) {
+
+    return {
+        LoadProductList: function() {
+            return jsonRpc.call(
+                    'product.product', 'scan_to_inventory_load_product', []).then(function (res) {
+                $rootScope.ProductListByEan13 = res;
+                var quantity = 0;
+                for (var a in res){quantity ++;}
```

Comment:
Because it doesn't work. 
**log**
![console_screen](https://cloud.githubusercontent.com/assets/3407482/20263913/32fb7c0e-aa69-11e6-9ee9-b5cffd35333e.png)

**result**
![log_screen](https://cloud.githubusercontent.com/assets/3407482/20263910/310dc2bc-aa69-11e6-8f20-39f2aa1b857b.png)

Any idea ?
(indeed, It's not very clean)